src/FffProcessor.cpp

```diff
@@ -135,6 +135,9 @@ bool FffProcessor::processMeshGroup(MeshGroup* meshgroup)
         gcoder.writeGCode();
         log("finished Neith Gcode generation...\n");
         
+    } else if (meshgroup->getSettingBoolean("lenser_backend_enabled"))
```

Comment:
I find it a bit weird to log this message here. Maybe you should log it at the place you use the setting.